benchmarks/benchmark_registration.py

```diff
@@ -1,14 +1,77 @@
-from skimage.data import stereo_motorcycle
+import numpy as np
+from scipy import ndimage as ndi
+
 from skimage.color import rgb2gray
-from skimage import registration
+from skimage import data, img_as_float
+
+# guard against import of a non-existant registration module in older skimage
+try:
+    from skimage import registration
+except ImportError:
+    pass
+
+# deal with move and rename of phase_cross_correlation across versions
+try:
+    from skimage.registration import phase_cross_correlation
+except ImportError:
+    try:
+        from skimage.feature import register_translation
+        phase_cross_correlation = register_translation
+    except ImportError:
+        phase_cross_correlation = None
 
 
 class RegistrationSuite(object):
     """Benchmark for registration routines in scikit-image."""
     def setup(self):
-        I0, I1, _ = stereo_motorcycle()
+        try:
+            from skimage.registration import optical_flow_tvl1
+        except ImportError:
+            raise NotImplementedError("optical_flow_tvl1 unavailable")
+        I0, I1, _ = data.stereo_motorcycle()
         self.I0 = rgb2gray(I0)
         self.I1 = rgb2gray(I1)
 
     def time_tvl1(self):
-        registration.tvl1(self.I0, self.I1)
+        registration.optical_flow_tvl1(self.I0, self.I1)
+
+
+class PhaseCrossCorrelationRegistration:
+    """Benchmarks for feature.register_translation in scikit-image"""
```

Comment:
reword this docstring to mention the new API instead?

Reply:
done

Reply:
updated in c73f686 to set the function to `skeletonize_3d` instead for skimage < 0.16.0